rpcs3/Emu/RSX/Program/GLSLCommon.cpp

```diff
@@ -681,8 +681,14 @@ namespace glsl
 			"{\n"
 			"	vec4 low = cl * 12.92;\n"
 			"	vec4 high = 1.055 * pow(cl, vec4(1. / 2.4)) - 0.055;\n"
-			"	bvec4 select = lessThan(cl, vec4(0.0031308));\n"
-			"	return clamp(mix(high, low, select), 0., 1.);\n"
+      #ifdef __APPLE__
+            // temporary workaround for moltenVK
+			"	bvec4 sel = lessThan(cl, vec4(0.0031308));\n"
+			"	return clamp(mix(high, low, sel), 0., 1.);\n"
```

Comment:
I think all this shielding around this code is unjustified. It's simply a variable rename, it shouldn't cause any harm to other platforms.

Reply:
Its a bug workaround. The variable can be omitted entirely by the time we're committing this, its just there because a bug report is open to track this externally I believe.

Reply:
It looks like a bug in SPIRV-Cross, but I haven't reported it yet because I wanted to fix it myself to be 100% sure, and also because I wanted to try more games.

Reply:
It should be possible to reproduce using a trivial shader and send it to them. Just something a few lines long, I don't think it needs to be some complex rpcs3 shader to show the problem. They are usually quite fast at fixing these kinds of things.

Reply:
Just rename.
Upstream bugs are not a concern for our code.
It suffices if our code compiles and the upstream repo maintained are notified about the issue.